util/middleware.js

```diff
@@ -9,30 +9,34 @@ export function getUserUrl (type) {
 }
 
 /**
- * This convenience method wraps a fetch call to the specified URL
- * with the token and api key added (if provided) to the HTTP request header.
+ * This method obtains an Auth0 token and passes it as header, along with the app's API key,
+ * to the fetch method using the provided URL and parameters.
  * @param {string} url The URL to call.
- * @param {string} accessToken If non-null, the Authorization token to add to request header.
+ * @param {string} auth0 the auth0 object used to obtain an accessToken for secure APIs.
  * @param {string} method The HTTP method to execute.
  * @param {*} options Extra fetch options to pass to fetch.
+ * @returns An object with the operation's status and data or error.
  */
-export async function secureFetch (url, accessToken, method = 'get', options = {}) {
-  const res = await fetch(url, {
-    method,
-    headers: {
-      Authorization: `Bearer ${accessToken}`,
-      'x-api-key': process.env.API_KEY
-    },
-    ...options
-  })
-  const json = await res.json()
-  return json
-}
+export async function secureFetch (url, auth0, method = 'get', options = {}) {
+  console.log(`Received call to secureFetch with params`, url, method, options)
```

Comment:
Remove console.log

Reply:
Thanks for catching that! https://github.com/ibi-group/otp-admin-ui/pull/33/commits/23b62595b7007cd4240e2a3adb36f93143a05a9a